packages/circuit-ui/components/Avatar/Avatar.tsx

```diff
@@ -55,10 +56,15 @@ const avatarSizes = {
 };
 
 const placeholders = {
-  object:
-    '<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 96 96"><path fill="white" d="M30 25c0-4.9706 4.0294-9 9-9s9 4.0294 9 9-4.0294 9-9 9-9-4.0294-9-9zM41.1571 60.5691L30.6742 48.3905c-1.6438-1.9097-4.6225-1.8422-6.1782.1399L8 69.5483v12.4515c0 3.3137 2.6863 6 6 6h5.9592l21.1979-27.4307zM70.4856 32.878c1.5553-2.002 4.5569-2.0705 6.202-.1417l11.312 13.2623v36c0 3.3137-2.6863 6-6 6H27.6611L70.4856 32.878z"/></svg>',
-  identity:
-    '<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 96 96"><path fill="white" d="M48 18c-7.732 0-14 6.268-14 14s6.268 14 14 14 14-6.268 14-14-6.268-14-14-14zM47.9998 88C61.53 88 73.4913 81.2822 80.73 71c-7.2387-10.2822-19.2-17-32.7303-17-13.5302 0-25.4914 6.7178-32.7302 17 7.2388 10.2822 19.2 17 32.7303 17z"/></svg>',
+  object: (
+    <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 96 96">
+      <path
+        fill="currentColor"
+        d="M30 25c0-4.9706 4.0294-9 9-9s9 4.0294 9 9-4.0294 9-9 9-9-4.0294-9-9zM41.1571 60.5691L30.6742 48.3905c-1.6438-1.9097-4.6225-1.8422-6.1782.1399L8 69.5483v12.4515c0 3.3137 2.6863 6 6 6h5.9592l21.1979-27.4307zM70.4856 32.878c1.5553-2.002 4.5569-2.0705 6.202-.1417l11.312 13.2623v36c0 3.3137-2.6863 6-6 6H27.6611L70.4856 32.878z"
+      />
+    </svg>
+  ),
```

Comment:
An icon from the icon library should replace this SVG. We might need to add the relevant icon to the library first.

Reply:
Yes. [Icon ref](https://www.figma.com/file/tEzkJmQSnX9bBw3mHtpuT1/%E2%9A%A0%EF%B8%8F-Circuit-UI-Mobile?node-id=28%3A7216&t=uqBCOM4JsVYv2nOx-4) (internal link)

Reply:
I added the icon and then realized we had it already 🙈 

38b01f9